ForceFeedback/xUnitTest/ConfigPathTest.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace xUnitTest
+{
+    public class ConfigPathTest
+    {
+		[Theory]
+		//[InlineData("I:\\Work\\49_c# algorithm\\Source\\ThreeProb\\ThreeProb\\HashIterator.cs")]
+		[InlineData("C:\\projects\\forcefeedbackprogramming\\ForceFeedback\\ForceFeedback.Rules\\Configuration\\Global.cs")]
+		public void FindProjectOrSolutionTest(string path)
```

Comment:
Please add test cases! Also an "or" in a test name does not make it clear what gets tested.
There should be test cases at least for:
* no config file is found
* a config file is found for the project
* a config file is found for the solution
* there are two config files (eg project + global)

All test cases should be coded as separate test functions.

All test cases must avoid absolute paths (implicit or explicit).

All test cases must be able to run independently.

Reply:
i will add 4 solutions (each solution for each case) for test in FFP. ok?